x-pack/plugins/security/public/management/role_mappings/edit_role_mapping/mapping_info_panel/mapping_info_panel.tsx

```diff
@@ -10,6 +10,7 @@ import {
   EuiTitle,
   EuiText,
   EuiSpacer,
+  // @ts-ignore
```

Comment:
__note:__ for some reason our custom typings for this and `EuiCodeEditor` aren't picked up in NP plugins when you run `type_check` - trying to figure this out.

Reply:
Created PR https://github.com/elastic/kibana/pull/55042